Java studypage buttons

VeeNinetyFour opened the pull request:
Includes temporary button tho the parts of the app that we weren't able to access before without modifying AndroidManifest. Will make testing a lot easier.

VeeNinetyFour committed:
Java studypage

Includes temporary button tho the parts of the app that we weren't able to access before without modifying AndroidManifest. Will make testing a lot easier.

VeeNinetyFour committed:
Minor aesthetic tweak

avgog reviewed (approved):
De knoppen werken goed, dus ik approve het. Graag wil ik wel dat je code in StudyProgramsActivity.java korter maakt (door bijv een for loop te gebruiken) als dat mogelijk is.

VeeNinetyFour committed:
Update strings.xml

avgog committed:
StudyCourse page

for loop voor het toevoegen van onclicklisteners gemaakt om de
code korter te maken.

avgog committed:
Merge branch 'JAVA-STUDYPAGE' of https://github.com/Veldhuis94/OPEN-DAY-APP into JAVA-STUDYPAGE

avgog reviewed (approved):
Code van de Study course page voor Bastiaan aangepast (zie mijn vorige review).
Was al tevreden met de rest van de pull request.

agrida2 reviewed (approved):
Deleted a unused button in java and xml. And added a multiple button usage in for loop nice way of doing it!!!

Mohamed1651 reviewed (approved):
Gr8 job! No errors.
